### PR TITLE
Add reusable badge component

### PR DESCRIPTION
**Pull request by jessethach**
## Changes

- Add component for badges
- Add styling
- Add icons to project from Figma

## Screenshots
Screen Shot 2022-11-10 at 12 02 16 PM

## Checklist

- [ ] if adds a new page, adds accessibility tests
- [ ] if adds a new page or new interaction, adds e2e test
- [ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)
- [ ] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)

Fixes https://app.asana.com/0/1203295759985135/1203326776555909

### CODE REVIEW

**Review by jlhogan — APPROVED** [review]
LGTM !

File: src/components/ResultDetail/BasicResultDetail.tsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ import { ReactComponent as Website } from "../../images/website.svg";
 import { ReactComponent as Location } from "../../images/location.svg";
 import { ReactComponent as DollarSign } from "../../images/dollar-sign.svg";
 import { ReactComponent as Clock } from "../../images/clock.svg";
+// TODO: wire in badges to list
+// import { ReactComponent as BadgeCheck } from "../../images/badge-check.svg";
+// import { ReactComponent as Telehealth } from "../../images/telehealth.svg";
```
Comment (SMakaiTakori) [inline src/components/ResultDetail/BasicResultDetail.tsx:11]: This is just a note for future implementation, I assume?

**Review by SMakaiTakori — APPROVED** [review]
Looks great!

File: src/components/ResultDetail/BasicResultDetail.tsx (hunk 1 above)
Comment (jessethach) [reply src/components/ResultDetail/BasicResultDetail.tsx:11]: Yes!

**Pull request merged**